src/classes/Validator.js

```diff
@@ -16,16 +16,27 @@ export default class Validator {
    * @param  {string} className - a CSS class
    * 
    */
```

Comment:
From line 12 to line 18: that part should be before the `showAlert` method. `ShowAlert` it's a render function: a function that renders a piece of the UI (a view). It's ok.